bin/key_server.js

```diff
@@ -163,6 +163,12 @@ function main() {
       server.log.fatal(err)
       process.exit(8)
     })
+    process.on('unhandledRejection', (reason, promise) => {
```

Comment:
This is just moving this code from the log module to here, since we probably don't want this during tests, only when the server is fully running.